tortoise.py

```diff
@@ -1,8 +1,18 @@
+import sys
+
+
 def main():
-    FILENAME = 'tiny.tortoise'
+    
+    try:
+        FILENAME = sys.argv[1]
+    except IndexError:
+        print('\nYou have to specify what tortoise you want executed.\
+              \nPlease add it as an argument on the command line.\
+              \nYour choices are: tiny.tortoise, medium.tortoise, or large.tortoise.\n')
```

Comment:
I would remove this line. The user should be able to specify whichever file they like, they are not limited to these three. If the user wants to see what's available, that's what `ls` is for :)